homeassistant/components/mqtt/__init__.py

```diff
@@ -181,13 +194,13 @@ def mqtt_topic_subscriber(event):
                          event.data[ATTR_PAYLOAD], event.data[ATTR_QOS],
```

Comment:
The yield from above this add_job isn't needed, it should be scheduled since add_job doesn't wait for the reply.